spec/models/query_spec.rb

```diff
@@ -12,9 +11,27 @@
     let!(:results) { exercise.submit_query!(user, query: 'foo', content: 'bar', cookie: ['foo', 'bar']) }
     let(:assignment) { exercise.find_assignment_for(user, Organization.current) }
 
-    it { expect(results[:status]).to eq :passed }
-    it { expect(results[:result]).to eq '5' }
-    it { expect(assignment.solution).to eq 'bar' }
-    it { expect(assignment.status).to eq :pending }
+    shared_examples_for 'a query submission' do
+      it { expect(results[:status]).to eq :passed }
+      it { expect(results[:result]).to eq '5' }
+      it { expect(assignment.solution).to eq 'bar' }
+      it { expect(assignment.status).to eq :pending }
+    end
+
+    context 'a playground exercise' do
+      let(:exercise) { create(:playground) }
+
+      it_behaves_like 'a query submission'
+      it { expect(assignment.submission_id).to_not be nil }
+      it { expect(assignment.submitted_at).to_not be nil }
+    end
+
+    context 'a problem problem' do
```

Comment:
a problem problem :laughing:

Reply:
Ups

Reply:
It was very problematic :stuck_out_tongue: